### ISSUE TITLE
Start with Windows Not Working on Admin Account

### ISSUE THREAD

**Issue by shinra358** [PeterTh/gedosato#149]
Start with windows doesn't work in the standard account because of the UAC. Coding so that the prompt doesn't appear, to automatically run as admin, or make it so that it uses a shortcut in the startup folder with elevated priviledges to fix.

**Comment by shinra358** [comment]
I just switched to an admin account. I've had 3 other programs that didn't autostart on the standard account too.

Those other programs now autostart. Gedosato still doesn't autostart even if I schedule it to with the Task Scheduler on the admin account.

**Comment by SwinglesMan** [comment]
Still not working in 2015, even in admin account.

**Comment by karasuhebi** [comment]
>Gedosato still doesn't autostart even if I schedule it to with the Task Scheduler on the admin account.

If you have GeDoSaTo in the Program Files folder, take it out of there. I have mine in C:\GeDoSaTo and it auto-starts just fine with a scheduled task.

**Comment by shinra358** [comment]
Nah, it's just a standard account issue. I don't have it in program files. I've since moved to an admin account. But I have a proposal:


The way most companies get past this is to tell the program to create it's own task scheduler event to bypass the uac when start with windows is activated. CCleaner, MSI Afterburner, etc. all do this. Of course a standard account wouldn't allow it to work, but you shouldn't play on a standard windows account anyway if you're a gamer. Causes many confilcts with alot of things.
